guides/common/modules/proc_creating-a-compliance-policy.adoc

```diff
@@ -8,7 +8,7 @@ Use this procedure to create a compliance policy, which you can use to ensure th
 * You have available SCAP contents, and eventually tailoring files, in {Project}.
 ** To verify what SCAP contents are available, see xref:listing-available-scap-contents_{context}[].
 ** To upload SCAP contents and tailoring files, see xref:Configuring_SCAP_Contents_{context}[].
-* Your user account has the `view_policies` and `create_policies` permissions.
+* Your user account has a role assigned that has the `view_policies` and `create_policies` permissions.
```

Comment:
alternative: "a role assigned with X any Y persmissions"?

Reply:
That sounds a bit odd, sorry, going with "that has" :D